services/db-manager/src/main/resources/db/changelog/changes/005_create_instruments_table.yaml

```
@@ -0,0 +1,119 @@
+                  name: description
+                  type: TEXT
+              - column:
+                  name: asset_class
+                  type: asset_class_enum
+                  constraints:
+                    nullable: false
+              - column:
+                  name: currency_code # Base currency of the instrument
+                  type: VARCHAR(3)
+                  constraints:
+                    nullable: false 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)